### PR TITLE
feat: complete task

### PR DESCRIPTION
**Pull request by SamGamgy**

### CODE REVIEW

**Review by mikesz88 — APPROVED** [review]
Love it!

**Review by brawblan — CHANGES_REQUESTED** [review]
A PR was just merged, so you'll need to pull changes (git pull origin main) and then repush your changes.

**Review by mikesz88 — CHANGES_REQUESTED** [review]
Resolve Conflict

File: standards-task.html [hunk 1]
```diff
@@ -16,6 +16,8 @@ <h2>Standards Task</h2>
     <li>Michael Sanchez</li>
     <li>Greg Perry - Chizy21</li>
     <li>Clayton Cripe</li>
+    <li>Sam Elder</li>
+
```
Comment (mikesz88) [inline standards-task.html:20]: delete this extra break line on line 20.

**Review by mikesz88 — CHANGES_REQUESTED** [review]
please fix the small break line and it is good!

File: standards-task.html (hunk 1 above)
Comment (SamGamgy) [reply standards-task.html]: Got it

**Review by brawblan — APPROVED** [review]
rad

**Review by mikesz88 — APPROVED** [review]

**Pull request merged**